autorest/adal/token.go

```diff
@@ -916,6 +922,11 @@ func (spt *ServicePrincipalToken) refreshInternal(ctx context.Context, resource
 		v := url.Values{}
 		v.Set("client_id", spt.inner.ClientID)
 		v.Set("resource", resource)
+		if spt.enablePoP == true {
```

Comment:
can drop ` == true` since enablePoP is a bool

Reply:
Dropped